lbrynet/extras/daemon/Daemon.py

```diff
@@ -2598,8 +2598,8 @@ def jsonrpc_channel_list(self, account_id=None, page=None, page_size=None):
         elif old_txo.claim.is_signed and not clear_channel and not replace:
             channel = old_txo.channel
 
-        if 'fee_address' in kwargs:
-            self.valid_address_or_error(kwargs['fee_address'])
+        if 'fee_amount' in kwargs:
+            kwargs['fee_address'] = self.get_fee_address(kwargs, old_txo.claim.stream.fee.address or claim_address)
```

Comment:
If user is just updating the fee_address and not touching the fee_amount, this change would never validate that the fee_address is valid address.